data/ship builders jobs.txt

```diff
@@ -0,0 +1,134 @@
+# Copyright (c) 2019 by Peter van der Meer
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+mission "Escort Hauler prototype to formal dinner"
+	description "Escort the ship with the new cargo-systems to <destination> to allow it to deliver sushi to a formal party."
+	source
+		neighbor planet "Skymoot"
+		not planet "Skymoot"
+	destination "Skymoot"
+	npc accompany
+		government "Merchant"
+		ship "Hauler III" "Southbound C Test"
+	on offer
+		conversation
+			`An elegantly looking lady is approaching you.`
+			`	"Captain <last>, captain <last>! It is such an honour to meet you!"`
+			choice
+				`	"Wow, that's quite a greeting. Why is it an honour to meet me?."`
+				`	"Hello, nice to meet you too."`
+					goto "hello too"
+				`	"Sorry, I don't have time to talk. I'm really busy."`
+					decline
+			`	She briefly smiles and says: "Well, I actually don't know you, but I just read your name on one of the cargo-manifests. And my experience is that this greeting works best if you need something from a spaceship captain."`
+			label "hello too"
+			`The lady continues talking;`
```

Comment:
Changing line 32 to "She says" gives 2 lines where we would start with she (if the player asks why it is a honor). I changed the first one to "She smiles and replies," to avoid also repeating the says twice.